pgzero/actor.py

```diff
@@ -112,6 +112,11 @@ def __repr__(self):
             self.pos
         )
 
+    def __dir__(self):
+        standard_attributes = [key for key in self.__dict__.keys()
+            if not key.startswith("_")]
```

Comment:
If users assign their own attributes to actors then they should show up in dir. This includes attributes they set that begin with _.

Alternatively we could prevent them setting attributes that begin with _ (as these would be reserved for Actor's internal use).